src/Microsoft.Sbom.Api/Executors/ComponentDetectionBaseWalker.cs

```diff
@@ -79,13 +79,17 @@ public ComponentDetectionBaseWalker(
         cliArgumentBuilder.AddDetectorArg("SPDX22SBOM", "EnableIfDefaultOff");
         cliArgumentBuilder.AddDetectorArg("ConanLock", "EnableIfDefaultOff");
 
-        if (sbomConfigs.TryGet(Constants.SPDX22ManifestInfo, out var spdxSbomConfig))
+        // Iterate over all supported SPDX manifests and apply the necessary logic
+        foreach (var supportedSpdxManifest in Constants.SupportedSpdxManifests)
         {
-            var directory = Path.GetDirectoryName(spdxSbomConfig.ManifestJsonFilePath);
-            directory = fileSystemUtils.GetFullPath(directory);
-            if (!string.IsNullOrEmpty(directory))
+            if (sbomConfigs.TryGet(supportedSpdxManifest, out var spdxSbomConfig))
```

Comment:
Presumably we can only match 1 config--should we break out of the loop after the first match?

Reply:
I wasn't sure if the customer was allowed to pass in multiple manifests. Seems like the conclusion is that there is only matching manifest based on your comment and what I've seen in the rest of the code, so I'll add a break statement after a successful match